CMakeLists.txt

```
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.14)
+
+## Set Project and Version
+file(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/plugin_name.txt" PLUGIN_NAME)
+file(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/version.txt" VERSION)
+project("${PLUGIN_NAME}" VERSION "${VERSION}" LANGUAGES CXX)
+
+## Get CMaize
+include(cmake/get_cmaize.cmake)
+
+## Paths ##
+set(PLUGIN_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src")
+set(PLUGIN_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include")
+set(PLUGIN_TESTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/tests")
+
+## Options ##
+cmaize_option_list(
+    BUILD_TESTING OFF "Should the tests be built?"
+)
+
+## Find or build dependencies ##
+cmaize_find_or_build_dependency(
+    simde
+    URL github.com/NWChemEx/SimDE
+    VERSION master
+    BUILD_TARGET simde
+    FIND_TARGET nwx::simde
+    CMAKE_ARGS BUILD_TESTING=OFF
+)
+
+## Add libraries ##
+cmaize_add_library(
+    ${PROJECT_NAME}
+    SOURCE_DIR "${PLUGIN_SOURCE_DIR}/${PROJECT_NAME}"
+    INCLUDE_DIRS "${PLUGIN_INCLUDE_DIR}/${PROJECT_NAME}"
```

```suggestion
    SOURCE_DIR "${${PROJECT_NAME}_SOURCE_DIR}/${PROJECT_NAME}"
    INCLUDE_DIRS "${${PROJECT_NAME}_INCLUDE_DIR}/${PROJECT_NAME}"
```
Welcome to CMake meta-programming; I'm truly sorry you are here.